.github/workflows/build.yml

```diff
@@ -557,7 +557,7 @@ jobs:
         shell: bash
         run: |
           # NOTE: cmake does not support autodetection of OPENSSL_ROOT_DIR via brew
-          export OPENSSL_ROOT_DIR=/usr/local/opt/openssl
+          export OPENSSL_ROOT_DIR=/opt/homebrew/opt/openssl
```

Comment:
Slightly more robust might be using something like like `brew --prefix openssl`, rather than having the paths hardcoded.

Reply:
@Coeur can you apply the suggestion? (the CI is stuck anyway, due to lots of CI runs)

Reply:
Well, yes, but at the same time the above line says "macos-latest", which will always be on Apple Silicon from now on.
So, more robust... but for what? A path change in 20 years? Well, yes, I can do that.

Reply:
https://github.com/libevent/libevent/pull/1633/commits/e582b49faf307088bd951201d201f8d1246a1bd6

Reply:
> So, more robust... but for what?

The paths you previously had could already be considered incorrect, and were pointing at a different symlink location than given by --prefix.

Reply:
They are the same paths.